tests/__init__.py

```diff
@@ -21,4 +21,4 @@
 # attempt to do the patch before we load any synapse code
 do_patch()
 
-util.DEFAULT_TIMEOUT_DURATION = 20
+util.DEFAULT_TIMEOUT_DURATION = 20.0
```

Comment:
```suggestion
util.DEFAULT_TIMEOUT_DURATION = 20
```

Reply:
The variable should be a float, as per the documentation: https://twistedmatrix.com/documents/current/api/twisted.trial.util.html

Reply:
It is an unrelated change in this PR, please remove it.